src/main/java/nextstep/fp/StreamStudy.java

```diff
@@ -28,6 +28,13 @@ public static void printLongestWordTop100() throws IOException {
         List<String> words = Arrays.asList(contents.split("[\\P{L}]+"));
 
         // TODO 이 부분에 구현한다.
+        words.stream()
+                .filter(word -> word.length() > 12)
+                .sorted((s1, s2) -> s2.length() - s1.length())
```

Comment:
`Comparator` 를 사용해 보는건 어떨까요? 
```suggestion
                .sorted(Comparator.comparing(String::length).reversed())
```